conda-forge.yml

```
@@ -1,9 +1,14 @@
 build_platform:
```

Looks like some Azure jobs are running out of disk space. Would suggest adding [this `azure` option]( https://conda-forge.org/docs/maintainer/conda_forge_yml/#azure ) (and re-rendering) to clean up disk space on Azure

```suggestion
azure:
  free_disk_space: true
build_platform:
```